x-pack/test/apm_api_integration/common/registry.ts

```diff
@@ -148,7 +147,7 @@ export function RegistryProvider({ getService }: FtrProviderContext) {
                 );
 
                 // sync jobs from .ml-config to .kibana SOs
-                await supertest.get('/api/ml/saved_objects/sync').set('kbn-xsrf', 'foo');
+                await ml.syncMlJobs();
```

Comment:
there is already a method for this
```suggestion
                await ml.syncSavedObjects();
```